### ISSUE TITLE
a11y: Author names should be available in the "latest updates" section of the homepage.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#782]
```
Hooray for not including the author's name in the JSON feed that's being loaded.
```

Original issue reported on code.google.com by `mkwst@google.com` on 27 Jan 2012 at 3:30
